OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/impl/executors/RefreshUserOperationExecutor.kt

```diff
@@ -117,11 +117,19 @@ internal class RefreshUserOperationExecutor(
                 subscriptionModel.carrier = subscription.carrier ?: ""
                 subscriptionModel.appVersion = subscription.appVersion ?: ""
 
-                // We only add a push subscription if it is this device's push subscription.
-                if (subscriptionModel.type != SubscriptionType.PUSH || subscriptionModel.id == _configModelStore.model.pushSubscriptionId) {
+                // We only add a non-push subscriptions
+                if (subscriptionModel.type != SubscriptionType.PUSH) {
                     subscriptionModels.add(subscriptionModel)
                 }
             }
+            var currentPushSubscriptionId = _configModelStore.model.pushSubscriptionId
```

Comment:
It's confusing to me what the difference between `current` and `my`  `PushSubscription`.